packages/webcert/src/feature/careProvider/CareProviderModal.tsx

```diff
@@ -37,7 +37,7 @@ const CareProviderModal: React.FC = () => {
   })
 
   const handleClose = () => {
-    dispatch(updateIsCareProviderModalOpen(false))
+    !chooseUnitLink && dispatch(updateIsCareProviderModalOpen(false))
```

Comment:
Test should be added for this

Reply:
Please check if the test I wrote looks ok?